src/org/usfirst/frc/team1076/robot/commands/TeleopCommand.java

```diff
@@ -0,0 +1,57 @@
+package org.usfirst.frc.team1076.robot.commands;
+
+import org.usfirst.frc.team1076.robot.Gamepad;
+import org.usfirst.frc.team1076.robot.Gamepad.GamepadAxis;
+import org.usfirst.frc.team1076.robot.subsystems.FrontBackMotors;
+import org.usfirst.frc.team1076.robot.subsystems.LeftRightMotors;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * Controls the robot using a joy stick controller.
+ * This command is intended to run continuously for the entire lifetime of the teleop mode.
+ */
+public class TeleopCommand extends Command {
+
+	FrontBackMotors frontBack;
+	LeftRightMotors leftRight;
+	Gamepad gamepad;
+	
+    public TeleopCommand(Gamepad gamepad, FrontBackMotors frontBack , LeftRightMotors leftRight ) {
+         requires(frontBack);
+         requires(leftRight);
+         this.gamepad = gamepad;
+         this.frontBack = frontBack;
+         this.leftRight = leftRight;
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	// Strafing
+    	double x = gamepad.getAxis(GamepadAxis.LeftX);
+    	double y = gamepad.getAxis(GamepadAxis.LeftY);
+    	frontBack.setSpeed(x);
+    	leftRight.setSpeed(y);
+    	
+    	// Rotation
```

Comment:
Rotation will require a bit of trickery with setting the speeds of the motors.

Keep in mind two things:

1. The motors should not be set to a speed above 1 or below -1

2. Rotation and strafing must both be possible at the same time.